src/atomate2/lobster/schemas.py

```diff
@@ -629,21 +743,268 @@ def from_directory(
             strongest_bonds_icohp_cation_anion=sb_icohp_ionic,
             strongest_bonds_icoop_cation_anion=sb_icoop_ionic,
             strongest_bonds_icobi_cation_anion=sb_icobi_ionic,
-            # include additional fields for all bonds
-            cohp_data=cohp_obj,
-            coop_data=coop_obj,
-            cobi_data=cobi_obj,
+            calc_quality_summary=calc_quality_summary,
+            calc_quality_text=" ".join(calc_quality_text),
             dos=dos,
             lso_dos=lso_dos,
             charges=charges,
             madelung_energies=madelung_energies,
             site_potentials=site_potentials,
             gross_populations=gross_populations,
             band_overlaps=band_overlaps,
+            # include additional fields for all bonds
+            cohp_data=cohp_obj,
+            coop_data=coop_obj,
+            cobi_data=cobi_obj,
         )
+
+        if save_cba_jsons:
+            cba_json_save_dir = dir_name / "cba.json.gz"
+            with gzip.open(cba_json_save_dir, "wt", encoding="UTF-8") as f:
+                f.write("[")
+                if (
+                    doc.lobsterpy_data_cation_anion is not None
+                ):  # check if cation-anion analysis failed
+                    lobsterpy_analysis_type = (
+                        doc.lobsterpy_data_cation_anion.which_bonds.replace("-", "_")
```

Comment:
Why are we doint this?

Reply:
to keep the keys consistent as in our manuscript for lightweight jsons

Reply:
Yeah. I see. Might be good to change it in general

Reply:
Yes. I think this is a weird fix.

Reply:
Okay will make the suggested change 😄